osu.Game/Graphics/UserInterface/OsuCheckbox.cs

```diff
@@ -18,6 +18,11 @@ public class OsuCheckbox : Checkbox
         public Color4 UncheckedColor { get; set; } = Color4.White;
         public int FadeDuration { get; set; }
 
+        /// <summary>
+        /// Whether to play sounds when the state changes as a result of user interaction.
+        /// </summary>
+        protected virtual bool PlaySoundsOnUserChange => true;
```

Comment:
What was the reasoning for this?

Reply:
Because it's already playing from the mods being selected.

Reply:
Does that mean it's also playing 30 times when you click select all?

Reply:
Originally it was, but see the changes to ModSelectOverlay.

Reply:
I see, thanks.